external_services/space_v2.rb

```diff
@@ -0,0 +1,67 @@
+require 'httparty'
```

Comment:
dlaczego trzymasz wiele wersji jednego zadania? przecież po to jest właśnie git :)

Reply:
zeby miec zawsze dostepne wszystkie wersje rozwiązania zadania 😄 bez koniecznosci odnajdywania ich w komitach, czy ma to sens? 😊

Reply:
myślę, że jednak po to mamy gita i lepiej trzymać tylko ostatnią wersję rozwiązania ;)